src/run/run.c

```diff
@@ -2075,8 +2075,24 @@ static int acquire_invocation_id(sd_bus *bus, const char *unit, sd_id128_t *ret)
                                 &error,
                                 &reply,
                                 "ay");
-        if (r < 0)
+        if (r < 0) {
+               /* Let's ignore connection errors. This might be caused by that the service manager is being
+                * restarted. Handle handle this gracefully. */
+                if (sd_bus_error_has_names(
```

Comment:
There're now three checks for this, so seems worth a small helper. bus_error_is_connection?